test/mocks/MockCentrifugeChain.sol

```diff
@@ -159,6 +159,17 @@ contract MockCentrifugeChain is Test {
         _execute(_message);
     }
 
+    function recoverTokens(address fromContract, address toUser, address token, uint256 amount) public {
```

Comment:
You haven't swapped the param names here, but are actually calling this with token before toUser  already from the test
```suggestion
    function recoverTokens(address target, address token, address to, uint256 amount) public {
```